spec/index.html

```diff
@@ -355,59 +355,61 @@ <h3>Blank nodes</h3>
   <section id="section-triple-terms">
     <h3>Triple terms</h3>
 
-    <p>As the basic unit of description in RDF, the triple is a simple
-    abstraction for stating a directed relationship between two resources.
-    Sometimes, we need to describe more detailed circumstances underlying such
-    relationships. While we could describe these using more sophisticated
-    modelling, there is a tension between simple, direct statements and more
-    complex, granular descriptions. This may require a trade-off between
-    directness and completeness.</p>
-
-    <p>To bridge this difference, we can use <a
-    data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a> as
-    <em>references</em> to simple statements. A triple term is a fixed object
-    that identifies a statement through its constituent subject, predicate and
-    object components.</p>
-
-    <p>This enables us to describe more concrete circumstances, such as events,
-    relationship entities or tokens in data sources, <em>in relation to</em>
-    the simple statements. That allows us to keep them as is, rather than
-    requiring a complete remodelling into more detailed structures, for the
-    sake of occasionally adding details.</p>
-
-    <p>Conceptually, we can <em>annotate</em> a simple statement with a
-    reference to something more detailed:</p>
-    <pre class="example nohighlight">
-      &lt;Bob&gt; &lt;is interested in> &lt;the Mona Lisa&gt; <sup><strong>~ &lt;Bob's interest></strong></sup> .
-    </pre>
+    <p>As the basic unit of description in RDF, the triple states a simple,
+    directed relationship between two resources. Sometimes, we need to describe
+    more detailed circumstances underlying such relationships. Conceptually, we
+    can <em>annotate</em> a statement with something more detailed:</p>
 
-    <p>This annotation refers to Bob's interest as a concrete circumstance, with
-    a date of origin:</p>
-    <pre class="example nohighlight">
-      &lt;Bob's interest> &lt;is an> &lt;Interest> .
-      &lt;Bob's interest> &lt;since> &lt;4th of October 1998> .
-    </pre>
+    <figure id="fig3">
+      <a href="example-triple-annotation.svg" data-include="example-triple-annotation.svg">
+      </a>
+      <figcaption>Informal triple annotation example: a concretization of a
+      statement described with its own characteristics.</figcaption>
+    </figure>
 
-    <p>Formally, an annotation is formed by two triples: the statement itself
-    and a <a data-cite="RDF12-CONCEPTS#dfn-reifying-triple">reifying
-    triple</a>. This expresses the simple relationship and the reifying relation to that from the circumstance, as a concretization thereof:</p>
+    <p>In conjunction to the arc expressing the relationship, the dashed circle
```

Comment:
```suggestion
    <p>In conjunction with the arc expressing the relationship, the dashed circle
```

Reply:
I figured using "to" was a bit more precise in this context, but (as a non-native speaker) I may have overlooked it just appearing less common. Presuming the latter, I'll accept this change.